### PR TITLE
Adding Travis-CI integration

### PR DESCRIPTION
**Pull request by yanosz**
Hello folks,

After some problems with building packages for 15.01 I decided to set up an CI based on Travis for the 15.05.1-branch. The result is here: https://travis-ci.org/yanosz/packages-1

Please include this configuration and Travis in your system, too - it's annoying to build non-building packages

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: bash
```
Comment (ecsv) [inline .travis.yml]: Cannot find "bash" in the language documentation of travis-ci. I would have expected c/cpp here

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+install:
```
Comment (ecsv) [inline .travis.yml]: Here is something missing to make sure that wget is the newest version from precise:

    sudo: required
    install:
      - sudo apt-get update && sudo apt-get install wget

Or just switch to trusty via:

    dist: trusty
    sudo: false

**Comment by lynxis** [comment]
@yanosz how about using the same script as https://github.com/openwrt/packages is using?

**Comment by PolynomialDivision** [comment]
> @yanosz how about using the same script as https://github.com/openwrt/packages is using?

Yeah! Awesome idea.

**Comment by yanosz** [comment]
> @yanosz how about using the same script as https://github.com/openwrt/packages is using?

Well, this PR is from 30 Jul 2016, packages starting using travis as of Jun 12, 2017.

I'm somewhat surprised, that this zombie-PR is alive, still. Nvm.

**Pull request closed without merging**